<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labelVideo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>1301</width>
     <height>721</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>310</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string> Video Stream</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>840</y>
     <width>682</width>
     <height>66</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>66</height>
    </size>
   </property>
   <property name="title">
    <string>Address:</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>E:\1.mp4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton1">
      <property name="text">
       <string>这是播放暂停按钮！不要问我为什么会在这里！</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonOpen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>850</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开视频路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>890</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置视频保存路径根目录</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>20</y>
     <width>161</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOpen">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>849</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSave">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>890</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\视频</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>760</y>
     <width>1281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>790</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存视频段</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditProcess">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>780</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBegin">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>790</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>442</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEnd">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>790</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>538</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>800</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>800</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>End:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>780</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>千分比</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDingWeiStart">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>790</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>定位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDingWeiEnd">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>790</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>定位</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
